assets/src/scripts/components/hydrograph/index.js

```diff
@@ -49,21 +51,93 @@ const plotDataLine = function (elem, {visible, lines, tsDataKey, xScale, yScale}
         .y(d => yScale(d.value));
 
     for (let line of lines) {
-        elem.append('path')
-            .datum(line.points)
-            .classed('line', true)
-            .classed('approved', line.classes.approved)
-            .classed('estimated', line.classes.estimated)
-            .attr('data-title', tsDataKey)
-            .attr('id', `ts-${tsDataKey}`)
-            .attr('d', tsLine);
+        if (line.classes.dataMask === null) {
+            elem.append('path')
+                .datum(line.points)
+                .classed('line', true)
+                .classed('approved', line.classes.approved)
+                .classed('estimated', line.classes.estimated)
+                .attr('data-title', tsDataKey)
+                .attr('id', `ts-${tsDataKey}`)
+                .attr('d', tsLine);
+        }
+        else {
+            let maskCode = line.classes.dataMask.toLowerCase();
```

Comment:
We seem to be using let in places where we should use const. Except for maskGroup, I think these all should be const.

Reply:
Okay.